bin/headers-staging.lambda.js

```diff
@@ -2,10 +2,10 @@ const api = 'https://staging-eapd-api.cms.gov';
 const okta = 'https://impl.idp.idm.cms.gov';
 const ga = 'https://www.google-analytics.com';
 const gtm = 'https://www.googletagmanager.com';
-const gf = 'https://font.googleapis.com';
+const gf = 'https://fonts.googleapis.com';
 
 const securityHeaders = {
-  'Content-Security-Policy': `default-src 'self' 'unsafe-inline'; connect-src ${api} ${okta}; child-src ${okta}; frame-ancestors ${okta}; img-src 'self' ${api} ${okta} ${ga}; object-src 'none'; script-src ${gtm} ${ga} ${okta}; style-src ${gf}`,
+  'Content-Security-Policy': `default-src 'self' 'unsafe-inline'; connect-src ${api} ${okta}; child-src ${okta}; frame-ancestors ${okta}; img-src 'self' ${api} ${okta} ${ga}; object-src 'none'; script-src 'self' ${gtm} ${ga} ${okta}; style-src 'self' 'unsafe-inline' ${gf}`,
```

Comment:
Does there need to be a trailing ; after the ${gf} and the `, It looks like in previous iterations there was